website/content/en/pre-docs/getting-started/_index.md

```diff
@@ -229,6 +229,7 @@ spec:
       labels:
         app: inflate
     spec:
+      terminationGracePeriodSeconds: 0
```

Comment:
Isn't this equivalent to force deletion on pods? Do we want this?

Reply:
this is just for the demo pause pods, so should be fine and might scale down faster